peer.js

```diff
@@ -692,9 +689,39 @@ function _maybeInvalidateScore(reason) {
     }
 };
 
+TChannelPeer.prototype.getScoreRange = function getScoreRange() {
+    var self = this;
+
+    if (!self._range) {
```

Comment:
Let's not add caching in this pr please, defer for later work.

Also, if we do go with this mode of caching, it should be contained in the getScoreRange method entirely:
- computeScoreRange should just compute and return
- the set of self._range should be in getScoreRange